pom.xml

```diff
@@ -475,7 +476,8 @@
                                 <licenseMerge>MIT License|MIT|The MIT License|MIT license|The MIT License (MIT)</licenseMerge>
                                 <licenseMerge>
                                     BSD 3-Clause License|BSD License|BSD|BSD licence|The BSD License|3-Clause BSD License|
-                                    New BSD License|New BSD license|The New BSD License|Revised BSD|BSD New license
+                                    New BSD License|New BSD license|The New BSD License|Revised BSD|BSD New license|
+                                    BSD-3-Clause
```

Comment:
Isn't `BSD 3-Clause License` and `BSD-3-Clause` the same? Just checking...

Reply:
Yup, but IIRC one of the dependencies has this exact string `BSD-3-Clause`. Hence this extension.